docker/Dockerfile

```diff
@@ -42,6 +42,20 @@ RUN pip3 install -r requirements.txt
 COPY requirements-wheels.txt /requirements-wheels.txt
 RUN pip3 wheel --wheel-dir=/wheels -r requirements-wheels.txt
 
+# Download and Convert OpenVino model
+FROM debian:11 as ov-converter
+RUN apt-get update && apt-get install -y wget ffmpeg libsm6 libxext6 \
```

Comment:
why is ffmpeg needed here?

Reply:
Good question.  I think that was carried over from one of my dev images.  I'll review the packages included in the converter stage and the final image and try to pare it back.

Reply:
I updated the packages for this converter stage and merged the intel-opencl-icd package into the arm64-only section of the final stage.